<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIBackupRam</class>
 <widget class="QDialog" name="UIBackupRam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Backup Ram Manager</string>
  </property>
  <layout class="QGridLayout">
   <item row="2" column="0" colspan="6">
    <widget class="QDialogButtonBox" name="dbbButton">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QGroupBox" name="gbSaveInformations">
     <property name="title">
      <string>Save Information</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="lFileName">
        <property name="text">
         <string>File Name :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leFileName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lComment">
        <property name="text">
         <string>Comment :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leComment">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lLanguage">
        <property name="text">
         <string>Language :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leLanguage">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lDataSize">
        <property name="text">
         <string>Data Size :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leDataSize">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lBlockSize">
        <property name="text">
         <string>Block Size :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leBlockSize">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lDate">
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leDate"/>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>120</number>
     </property>
     <property name="bottomMargin">
      <number>120</number>
     </property>
     <item>
      <widget class="QPushButton" name="pbCopyFromLocal">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCopyFromCloud">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gbSaveList">
     <property name="minimumSize">
      <size>
       <width>224</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Local Save List</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QListWidget" name="lwSaveList"/>
      </item>
      <item>
       <widget class="QLabel" name="lBlocks"/>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="pbDelete">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbFormat">
          <property name="text">
           <string>Format</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pbExport">
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbImport">
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QGroupBox" name="gbDeviceList">
     <property name="title">
      <string>Device List</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QComboBox" name="cbDeviceList"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>224</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Cloud Save List</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="lwCloudSaveList"/>
      </item>
      <item>
       <widget class="QPushButton" name="pbDeleteCloudItem">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dbbButton</sender>
   <signal>rejected()</signal>
   <receiver>UIBackupRam</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
